Python-Function-ID.md

```diff
@@ -0,0 +1,42 @@
+# Python id(object)
+
+`id()` is a built-in function in Python 3, which returns the *identity* of an object. The *identity* is a unique integer for that object during its lifetime. In CPython implementation, this is the address of the object in memory which is not necessarily true for other python implementations.
```

Comment:
> which is not necessarily true for other python implementations.

I know that CPython is not a perform-ant implementation. PyPy and Pyston would probably handle performance better by moving around objects in memory.

But reading it, it feels like that `id()` need to be same, is not part of the spec. Is that so? I am not fully aware of this.